documentation/CLI-for-JFrog-Curation.md

```diff
@@ -0,0 +1,81 @@
+JFrog CLI : CLI for JFrog Curation
+======================================
+
+
+Overview
+--------
+
+This page describes how to use JFrog CLI with JFrog Curation.
```

Comment:
Let's add some basic information about JFrog Curation, or at least a link to a documentation page that include it.